cassandra/2-SubArrayWithMaxSum.cql

```diff
@@ -10,7 +10,7 @@ INSERT INTO devgym.task (
 ) VALUES (
   '2016-01-01 00:00:00 UTC',
   'scalaClass',
-  now(),
+  9894cd10-ce12-11e5-8ee9-091830ac5256,
```

Comment:
Used in ApplicationTest.scala

val Some(result) = route(FakeRequest(GET, s"/task/scalaClass/${Date.from(instant).getTime}/9894cd10-ce12-11e5-8ee9-091830ac5256"))